lib/features/composer/data/repository/composer_repository_impl.dart

```diff
@@ -58,9 +61,7 @@ class ComposerRepositoryImpl extends ComposerRepository {
       emailAttachments.addAll(tupleContentInlineAttachments.value2);
     }
 
-    if (createEmailRequest.draftsMailboxId == null) {
-      emailContent = await _removeCollapsedExpandedSignatureEffect(emailContent: emailContent);
-    }
+    emailContent = await _removeCollapsedExpandedSignatureEffect(emailContent: emailContent);
```

Comment:
Why ignore this condition?

Reply:
For this issue to be fixed
- https://github.com/linagora/tmail-flutter/issues/3038